docs/dataset/formatting-your-datasets.md

```diff
@@ -0,0 +1,162 @@
+| [`PointCloudAsset`](../reference/asset.md#kolena.asset.PointCloudAsset) | Useful for attaching 3D point cloud data.                      |
+
+### Kolena Annotations
+
+Kolena allows you to visualize overlays on top of datapoints through the use of[`annotation`](../reference/annotation.md).
+These annotations are visible on both the Gallery view for groups of datapoints and for individual datapoints.
+
+| Annotation Type                                                                      | Description |
+|--------------------------------------------------------------------------------------|----------------------------|
+| [`BoundingBox`](../reference/annotation.md#kolena.annotation.BoundingBox)            | Used to overlay bounding boxes (including confidence scores and labels) on top of images. |
+| [`SegmentationMask`](../reference/annotation.md#kolena.annotation.SegmentationMask)  | Used to overaly raster segmentation maps on top of images. |
+
+### Structured Data
+
+When uploading `.csv` files for datasets that contain annotations, assets or nested values in a column use the
+[`dataframe_to_csv()`](../reference/io.md#kolena.io.dataframe_to_csv) function provided by Kolena to save a `.csv` file
+instead of [`pandas.to_csv()`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html).
+
+In order to add data like a list of `BoundingBox` objects to your dataset via the sdk all you need to do is have
+field with a list of objects in your dataframe.
+
+A snippet like the following:
+
+```python
+from kolena.annotation import BoundingBox
+from kolena.io import dataframe_to_csv
+from collections import defaultdict
+import pandas as pd
+
+df = pd.read_csv(f"s3://kolena-public-examples/coco-2014-val/raw/coco-2014-val.csv", storage_options={"anon": True})
+image_to_boxes = defaultdict(list)
+image_to_metadata = defaultdict(dict)
+
+for record in df.itertuples():
+    coords = (float(record.min_x), float(record.min_y)), (float(record.max_x), float(record.max_y))
+    bounding_box = BoundingBox(*coords)
+    image_to_boxes[record.locator].append(bounding_box)
+    metadata = {
+        "locator": record.locator,
+        "height": record.height,
+        "width": record.width,
+        "date_captured": record.date_captured,
+        "brightness": record.brightness,
+    }
+    image_to_metadata[record.locator] = metadata
+
+df_boxes = pd.DataFrame(list(image_to_boxes.items()), columns=["locator", "ground_truths"])
+df_metadata = pd.DataFrame.from_dict(image_to_metadata, orient="index").reset_index(drop=True)
+df_merged = df_metadata.merge(df_boxes, on="locator")
+
+dataframe_to_csv(df_merged, "processed.csv")
+```
+
+### Formatting results for Object Detection
+
+For Object Detection problems, model results need to have the following columns
+for the best experience. The values for each of the columns is a [`List[ScoredLabeledBoundingBox]`](../reference/annotation.md#kolena.annotation.ScoredLabeledBoundingBox)
```

Comment:
I don't think we've tested it yet, but IIRC `ScoredLabeledPolygon` should also work

Reply:
Will verify and include.

Reply:
Since the[ matching logic ](https://github.com/kolenaIO/kolena/blob/trunk/kolena/_experimental/object_detection/dataset.py) uses the `top_left` and `bottom_right` fields from `LabeledBoundingBox` and `ScoredLabeledBoundingBox` this currently does not work (in terms of running the matching). We would need to expand the logic before we say this works for `ScoredLabeledPolygon`. I will create a ticket to track this follow up item.